booster_bdd/che.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+echo "Invoking Che test script here..."
+
+# -DosioUrlPart=$1           os.getenv("SERVER_ADDRESS")
+# -DosioUsername=$2          os.getenv("OSIO_USERNAME")
+# -DosioPassword=$3          os.getenv("OSIO_PASSWORD")
+# -DcheWorkspaceName=$4      workspaceName
+# -DkeycloakToken=$5         theToken
+
+set +x
+
+# Script to execute: che-functional-tests/cico/run_EE_suite.sh
+git clone git@github.com:redhat-developer/che-functional-tests.git
```

Comment:
Please, use shallow clone to save load, bandwidth and disk space: `git clone --depth 1 git@github.com:redhat-developer/che-functional-tests.git`